src/main/java/com/nooul/apihelpers/springbootrest/utils/StringUtils.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * https://github.com/Nooul/spring-boot-rest-api-helpers
+ * 
+ * Released under the MIT License.
+ * Please refer to LICENSE file for licensing information.
+ */
+
+package com.nooul.apihelpers.springbootrest.utils;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * String utilities
+ */
+public class StringUtils {
+
+    /**
+     * Convert a string to camel case
+     * 
+     * @param str
+     * @param capitalizeFirstLetter
+     * @param delimiters
+     * @return
+     */
+    public static String toCamelCase(String str, final boolean capitalizeFirstLetter, final char... delimiters) {
```

Comment:
why not use a library for that? apache common text is fine

Reply:
I try to use less library as possible for dependencies, this way you have less possibility of conflict. This is the reason why I "merged" part of Apache Common in code.

Reply:
I prefer less maintainence and easy upgradability which can't be done when I am using parts of a library or if I code it myself